lib/rex/ui/text/shell/history_manager.rb

```diff
@@ -0,0 +1,68 @@
+# -*- coding: binary -*-
+
+module Rex
+module Ui
+module Text
+module Shell
+
+class HistoryManager
+ 
+
+  @@contexts = [{"history_file" => Msf::Config.history_file, "name" => :msfconsole}]
```

Comment:
The `history_file` and `name` keys should be symbols.

Reply:
Done :+1: